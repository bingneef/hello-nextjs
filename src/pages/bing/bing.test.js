import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import Bing from "./bing";

describe("Bing", () => {
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Bing />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<Bing title="Bing" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    const rootInstance = component.root;
    const div = rootInstance.findByType("div");
    expect(div.props.children).toBe("Bing");
  });
});
