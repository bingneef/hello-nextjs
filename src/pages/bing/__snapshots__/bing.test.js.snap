// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bing matches the snapshot 1`] = `
<div
  className="root"
>
  Bing
</div>
`;
